R/utils.R

```diff
@@ -161,19 +193,30 @@ capture_expr_output <- function(expr, split = FALSE, env = parent.frame(),
   any_output_error <- any(vapply(outputs, inherits, logical(1L), "error"))
 
   structure(
-    if (!quoted) expr_quote else expr,
-    value = res$value,
-    visible = res$visible,
-    traceback = cnds_err_traceback,
-    output = outputs[nzchar(outputs)],
-    class = c("expr_output", "expression"))
+    res$value,
+    .recording = list(
```

Comment:
What does this '.recording' value do?

Reply:
.recording is a list of console output during the evaluation of the expression. Each line of stdout (from `cat`, `print`) is its own entry in the list, and each `message`, `warning`, or `error` gets its own entry. The print method prints these out as though we're playing back a recording of the execution in an interactive session. 

For now this is just used to capture errors, but we could use something like this to produce a trace of the steps that are executed to collect all the pkg_metric info if we want to generate a report.

I'm not sold on the complexity here, so if it ends up being problematic we can simplify it.